base/dates/periods.jl

```diff
@@ -358,8 +358,6 @@ GeneralPeriod = Union{Period, CompoundPeriod}
 
 for op in (:+, :-)
     @eval begin
-        ($op)(x::GeneralPeriod, Y::StridedArray{<:GeneralPeriod}) = broadcast($op, x, Y)
-        ($op)(Y::StridedArray{<:GeneralPeriod}, x::GeneralPeriod) = broadcast($op, Y, x)
```

Comment:
I think these would need deprecating too, GeneralPeriod isn't a subtype of Number right?